.devops/nix/package.nix

```diff
@@ -18,6 +18,8 @@
   vulkan-headers,
   vulkan-loader,
   curl,
+  shaderc,
+  clblast,
```

Comment:
What is CLBlast doing here?

Reply:
Ugh, looks like cruft pulled in from merging various conflicts in master in. Let me see where that happened.

Reply:
Yeah looks like I started on this change and shortly after the remnants of clblast were removed, then I accidentally kept it in when resolving conflicts.